person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_reader @id
   attr_accessor @name, @age
```

Comment:
- [ ] **[OPTIONAL]** I was unable to test your code because of these declarations here. It should be this:

  ```
  attr_reader :id
  attr_accessor :name, :age
  ```